src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -91,19 +92,21 @@ def start_workunit(self, workunit):
           sample_rate=self.sample_rate, # Value between 0.0 and 100.0
         )
         self.trace_id = zipkin_attrs.trace_id
-
+        self.parent_id = zipkin_attrs.span_id # temporary code
```

Comment:
It would be good to explain why it is temporary, and if it's something we'll want to fix, creating a TODO for it... it might not be clear to the reader _why_ it is temporary.